README.md

```diff
@@ -19,6 +19,7 @@ At the time of joining, the HR potrays the company like it is match made in Heav
 | 10 | Usually help half or whole financially for IT events and certifications?|
 | 11 | Can you describe the culture of the company?|
 | 12 | How easily the upper management is approachable for suggestion or queries?|
+| 19 | Can I work from home on Fridays?| 
```

Comment:
This question, might not be good for first impression towards the interviewer, could you please change the question.